src/tcgwars/logic/impl/gen8/ChillingReign.groovy

```diff
@@ -2288,13 +2289,14 @@ public enum ChillingReign implements LogicCardInfo {
             attackRequirement {}
             onAttack {
               damage 20
+              apply(POISONED)
```

Comment:
For future reference: special conditions applied via an attack should normally (if not always) use `applyAfterDamage`, `apply` is more for stuff like abilities and trainers.

```suggestion
              applyAfterDamage POISONED
```